### DISCUSSION TITLE
Unable to link Deepmd v3 to LAMMPS

### DISCUSSION THREAD

**Discussion by FarnesolSXS** [deepmodeling/deepmd-kit#4118] [Q&A]
Hi, I'm wondering how I can activate Deepmd module in LAMMPS. Both LAMMPS and Deepmd-kit v3 are shown as python packages and located in my "site-package" folder. Neither can I find "source" folder in Deepmd package, nor "src" folder in LAMMPS, thus I'm not able to do anything like creating USER-DEEPMD. It seems like LAMMPS and Deepmd-kit v3 are installed automatically as libraries instead of softwares. Both Deepmd and LAMMPS are working well separately, and "Deepmd" & "LAMMPS" are in pip list now. Hope to find a method to use deepmd pairstyle in LAMMPS under this situation, tks a lot!
<img width="243" alt="20240910211458" src="https://github.com/user-attachments/assets/b2ce849f-0ca8-4d2b-b603-e5212390d141">
<img width="245" alt="20240910211449" src="https://github.com/user-attachments/assets/b9dc6b6d-acb2-447e-8109-857a8481bd5c">

**Comment by Hu-Yuch** [comment]
The same problem
